nucliadb/nucliadb/ingest/processing.py

```diff
@@ -110,6 +115,12 @@ class PushPayload(BaseModel):
         default_factory=models.PushProcessingOptions
     )
 
+    vectorsets: List[RequestedVectorSets] = Field(
```

Comment:
@nuclia/learning this is a proposal on how nucliadb will tell you which embedding models you need to compute for the resource being processed.